### ISSUE TITLE
Remove constraints for indent and nickname

### ISSUE THREAD

**Issue by pocmo** [pocmo/Yaaic#200]
We have continuously been tweaking this. It's impossible to create a whitelist - every server has different constraints. Let's just remove all constraints and allow whatever the user enters (and just trim). Let the servers decide what it accepts.

**Comment by Croydon** [comment]
Yeah, please do this. I can't connect to my IRC bouncer currently, since I would need to enter as a nickname `loginname/network` and `/` are forbidden.
